bucket/webcamoid.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "9.0.0",
+    "description": "Full featured webcam suite",
+    "homepage": "https://webcamoid.github.io/",
+    "license": "GPL-3.0",
```

```suggestion
    "license": "GPL-3.0-or-later",
```
We use [SPDX license identifiers](https://spdx.org/licenses/) as the standard.